ui/src/components/terms-of-service-info/terms-of-service-info.module.scss

```
@@ -2,15 +2,15 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
-  @include body-small();
+  @include body-small;
   padding: 8px 16px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)